Model/Source/AbstractInsurancePossibilities.php

```diff
@@ -27,15 +43,30 @@ public function toOptionArray(): array
      */
     public function toArray(): array
     {
-        return array_reduce($this->getInsurancePossibilitiesArray(), static function ($array, $insuranceValue) {
+        $cc = null;
+        if ($this->type === 'local') {
```

Comment:
dit is echt niet nice hier, als dit per se in deze class moet dan tenminste in een eigen functie

Reply:
Duidelijk, ik heb de betreffende logica verplaatst naar zijn eigen class.